src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -209,6 +211,10 @@ private static VersionInfo GetVersionInfo()
 
         public override ISchema Schema { get { return _bindings; } }
 
+        public bool CanSaveOnnx => _mapper is ISaveAsOnnx onnxMapper ? true : false;
```

Comment:
>_mapper is ISaveAsOnnx onnxMapper ? true : false; [](start = 35, length = 49)

Once you correct the interface this could be:

`(_mapper as ISaveAsOnnx)?.CanSaveOnnx ?? false`

See other instances of `CanSaveOnnx` to see this pattern.

Moot point, note that if you have an expression of the form `<boolExpr> ? true : false` this can be simplified to `<boolExpr>`.